README.md

```diff
@@ -14,8 +14,9 @@ and provide a smoother experience for people wishing to invest themselves in the
 * Yarn 1.22 or higher
 
 *Note*
-There is a `shell.nix` file provided for convenience. However it is far from perfect. Notably, it does not install
-GHC 8.10.4, nor manages a local installation of PostgreSQL (yet). Don't load this shell file with `--pure`.
+There is a `shell.nix` file provided for convenience. However it is far from perfect.
+It will ont manages a local installation of PostgreSQL for you (yet).
```

Comment:
```suggestion
It will not manage a local installation of PostgreSQL for you.
```

Reply:
Nix shell actually can't install postgres for you unless you want to start with a fresh postgres everytime you enter nix-shell (which is generally infeasible). We can direct people to this config though:

```nix
  services.postgresql = {
    package = pkgs.postgresql_12;
    enable = true;
    enableTCPIP = false;
    authentication = ''
      local all all trust
      host all all 127.0.0.1/32 trust
      host all all ::1/128 trust
    '';
    settings = {
      timezone = "UTC";
      shared_buffers = 128;
      fsync = false;
      synchronous_commit = false;
      full_page_writes = false;
    };
  };
```

Reply:
this is only for NixOS (maybe nix-darwin too)

Reply:
Yeah, IHP handles the postgres startup from the nix env too. I just wanted to clarify the limits of the nix shell. :)